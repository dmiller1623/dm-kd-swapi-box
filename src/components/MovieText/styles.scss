.movie-text {
  width: 25%;
  height: 100vh;
  position: fixed;
  color: #ddbe24;
  font-weight: bolder;
  text-align: center;
  background-image: url(https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQsWrlqShOpNL4SbAwB24CkJuGgezSJoX5KDsQDGPvTWpa-MtZ8mw)
}

.crawl-text {
  font-size: 35px;
}