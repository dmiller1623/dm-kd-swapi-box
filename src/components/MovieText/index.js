import React from 'react'
import './styles.css'
 
const MovieText = ({ movies }) => {
  return(    
      <marquee direction='up' scrollspeed='20'>
        <p>{movies.opening_crawl}</p>
        <h3>{movies.title}</h3>
        <h3>{movies.release_date}</h3>
      </marquee>
  )
}

export default MovieText