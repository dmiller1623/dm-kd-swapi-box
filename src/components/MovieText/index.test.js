import React from 'react'
import MovieText from './index.js'
import { shallow } from 'enzyme'

describe('MovieText', () => {
  it('should match the snaphot', () => {
    let mockMovie = { title: 'blaw', opening_crawl: 'sdklncknd', release_date: 38932}
    let wrapper = shallow(<MovieText movies={mockMovie}/>)
    expect(wrapper).toMatchSnapshot()
  })
})