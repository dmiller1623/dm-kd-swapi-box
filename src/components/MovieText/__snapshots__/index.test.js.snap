// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieText should match the snaphot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MovieText
    movies={
        Object {
            "opening_crawl": "sdklncknd",
            "release_date": 38932,
            "title": "blaw",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="crawl-text"
>
          sdklncknd
</p>,
        <h3>
          blaw
</h3>,
        <h3>
          38932
</h3>,
      ],
      "className": "movie-text",
      "direction": "up",
      "scrollspeed": "20",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "sdklncknd",
          "className": "crawl-text",
        },
        "ref": null,
        "rendered": "sdklncknd",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "blaw",
        },
        "ref": null,
        "rendered": "blaw",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 38932,
        },
        "ref": null,
        "rendered": 38932,
        "type": "h3",
      },
    ],
    "type": "marquee",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="crawl-text"
>
            sdklncknd
</p>,
          <h3>
            blaw
</h3>,
          <h3>
            38932
</h3>,
        ],
        "className": "movie-text",
        "direction": "up",
        "scrollspeed": "20",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "sdklncknd",
            "className": "crawl-text",
          },
          "ref": null,
          "rendered": "sdklncknd",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "blaw",
          },
          "ref": null,
          "rendered": "blaw",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 38932,
          },
          "ref": null,
          "rendered": 38932,
          "type": "h3",
        },
      ],
      "type": "marquee",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
